f8a_report/report_helper.py

```diff
@@ -565,13 +565,17 @@ def retrieve_worker_results(self, start_date, end_date, id_list=[], worker_list=
         ids = sql.SQL(', ').join(id_list).as_string(self.conn)
 
         for worker in worker_list:
-            query = sql.SQL('SELECT {} FROM {} WHERE {} IN (%s) AND {} = \'%s\'').format(
+            query = sql.SQL('SELECT {} FROM {} WHERE {} IN (%s) AND {} = \'%s\' AND '
+                            '\'_audit->version\' = \'v1\'').format(
                 sql.Identifier('task_result'), sql.Identifier('worker_results'),
                 sql.Identifier('external_request_id'), sql.Identifier('worker')
             )
 
             self.cursor.execute(query.as_string(self.conn) % (ids, worker))
             data = json.dumps(self.cursor.fetchall())
+            if not self.cursor.rowcount:
+                logger.info('No Data has been found for v1 stack analyses.')
+                return result_interim
```

Comment:
Do we really need these logs? What would happen if you don't handle `if not self.cursor.rowcount` case?

Reply:
@arajkumar Initally we have Route like--
Find Stack Analyses Ids in last 1 day (v1+v2) -> Find Worker results corresponding to that Ids.
This works fine as we have only v1 APIs.

Now We have:
Find Stack Analyses Ids in Last 1 Day (v1+v2) -> Find Worker results of v1 API only.

If we follow the previous approach and don't put any conditional checks here..
In case when v1 APIs execution is Zero we will have a crash ZeroDivisionError here..

https://github.com/fabric8-analytics/f8a-stacks-report/blob/5491baf377d1bd29b5e74f2c2b8f123eb02df41b/f8a_report/report_helper.py#L540

Reply:
@arajkumar Initally we have Route like--
Find Stack Analyses Ids in last 1 day (v1+v2) -> Find Worker results corresponding to that Ids.
This works fine as we have only v1 APIs.

Now We have:
Find Stack Analyses Ids in Last 1 Day (v1+v2) -> Find Worker results of v1 API only.

If we follow the previous approach and don't put any conditional checks here..
In case when v1 APIs execution is Zero we will have a crash ZeroDivisionError here..

https://github.com/fabric8-analytics/f8a-stacks-report/blob/5491baf377d1bd29b5e74f2c2b8f123eb02df41b/f8a_report/report_helper.py#L540